### PR TITLE
Add gem 'capistrano-rvm' as option to rbenv

### PR DESCRIPTION
**Pull request by adisos**
It's useful for those who are using rvm instead of rbenv.

### CODE REVIEW

File: Gemfile [hunk 1]
```diff
@@ -69,6 +69,7 @@ group :development do
   gem 'capistrano-rails',   require: false
   gem 'capistrano-rbenv',   require: false
   gem 'capistrano3-puma',   require: false
+  gem 'capistrano-rvm',     require: false, github: 'capistrano/rvm'
```
Comment (mfrederickson) [inline Gemfile]: I'm curious why you are specifying the github repo.

**Comment by stevecrozz** [comment]
Is rbenv not working for you @adisos?

File: Gemfile (hunk 1 above)
Comment (adisos) [inline Gemfile]: Removed that line

**Comment by adisos** [comment]
@stevecrozz 

I think it's not that bad to have more options to deploy. I don't like rbenv, I'm using rvm all the time, so I think I'm not only one who's also using it.

**Comment by stevecrozz** [comment]
@adisos That makes sense. And I don't mind adding a reference to rvm in the deploy.example.rb file, but I'm not enthusiastic about adding a new Gemfile dependency for everyone.

Would you be ok with adding to the deploy.example.rb file something more like:
"For rvm, add capistrano-rvm to UserGemfile and see https://github.com/capistrano/rvm#usage"

We could always treat rbenv the same way if we feel the need.

**Comment by stevecrozz** [comment]
Rvm is a great tool, but I'm closing this because I don't think Errbit needs more options in this area. For others curious about this, I'd be more interested in advancing the container-based deployment workflow where all of these concerns are hidden away inside the container so that (hopefully) nobody has to worry about them.

**Pull request closed without merging**